### PR TITLE
Upload files improvements

### PR DESCRIPTION
**Pull request by mitjade**
- upload multiple files
- list uploaded files
- remove files
- download link

### CODE REVIEW

**Comment by mairh** [comment]
Nice. I was thinking maybe write a real life example. We could upload a profile image and save the url link with meta data to user profile?

**Comment by mitjade** [comment]
@mairh I would rather keep examples separate, since then you are tying `upload` and `user` example together.

**Comment by mairh** [comment]
Yeah I was wondering about the same point. Maybe we keep it separate

File: package.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
     "error-stack-parser": "^2.0.1",
     "expo": "^23.0.0",
     "express": "^4.16.2",
+    "filesize": "^3.5.11",
```
Comment (mairh) [inline package.json]: This one is not added to `webpackAppConfig.js`

Comment (mitjade) [reply package.json]: It is really small and can be used on all platforms, so I did not add it.

**Pull request merged**